src/main/webapp/scripts/perspective-api-service.js

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Calls the Perspective API
+ *  callback function is called after API call completed
+ */
+
+function perspectiveApiCall(comment, callback) {
```

Comment:
Not very familiar with practices on using callback. Where did you find examples like this one? Why not make it just a function that returns the toxicity and then let the caller to whatever they want with it?

https://stackoverflow.com/questions/950087/how-do-i-include-a-javascript-file-in-another-javascript-file

It seems that you need to export the method in this file by doing `export function getToxicity(comment) {...}` and then in another js file you can do `import { getToxicity } from '<path to perspective-api-service.js>'` and then now you can call getToxicity("this is a comment"). With this you wouldn't lead to load all js files in the DOM. Only the main js.

Please let me know if you found good practices that prefer otherwise :)

Reply:
Also, if you want to discuss this at a faster rate. Maybe we could have GVC talk. This reduces a lot of friction between PR comments. If we were working together physically this is something I would schedule some time to discuss. Throughout the summer, I would prefer we get in quick gvc chats to discuss these topics quicker.

Reply:
Zoe showed me how she used the `import { function } from '<path>'` method, but I don't think its suitable for this file because it makes the perspective.js file into a module, meaning that it's functions cannot be called from elsewhere in the code. Since getToxicity() may be called from many places such as the chrome extension, I think it's best to leave it as is.